packages/twenty-front/src/modules/object-metadata/hooks/useCreateOneObjectMetadataItem.ts

```diff
@@ -34,8 +33,20 @@ export const useCreateOneObjectMetadataItem = () => {
       variables: {
         input: { object: input },
       },
-      awaitRefetchQueries: true,
-      refetchQueries: [getOperationName(FIND_MANY_OBJECT_METADATA_ITEMS) ?? ''],
```

Comment:
I actually think it's fine keeping the refetch query but removing awaitRefetchQuery